### ISSUE TITLE
Adding as a pinning service 

### ISSUE THREAD

**Issue by SaintGenesius** [nftstorage/nft.storage#2295]
I am looking for a way to add NFT Storage as a pinning service to my IPFS. Required are the API Endpoint and secret access code. Is the secret access code the Bearer Token? If so, how can I go about doing this? Endpoint being https://api.nft.storage - I presume. Any help would be greatly appreciated.

Thank you.

**Comment by alanshaw** [comment]
Yep, take a look here https://nft.storage/docs/how-to/pinning-service/ the secret access code is the API key you can get from the site - https://nft.storage/manage/ - you need to request access first using the button on that page.
